src/main/webapp/app/exercises/programming/shared/lifecycle/programming-exercise-lifecycle.component.ts

```diff
@@ -94,10 +94,34 @@ export class ProgrammingExerciseLifecycleComponent implements OnInit, OnChanges
             // Changes from parent component are allowed but no cascading changes should be made in read-only mode.
             return;
         }
+        if (this.exerciseService.hasStartDateError(this.exercise)) {
+            this.updateStartDate(newReleaseDate);
```

Comment:
I realise this has been done before but do we rally want to change some other date on error? I think it's counter-intuitive as a user that when an error is shown, stuff changes automatically.

Reply:
This is the current behavior of the complete component.